src/gcp/cloudfunctions.js

```diff
@@ -167,6 +169,10 @@ function _updateFunction(options) {
     data.maxInstances = Number(options.maxInstances);
     masks.push("maxInstances");
   }
+  if (options.environmentVariables) {
```

Comment:
Same question as @mbleigh - does this overwrite any existing environment variables?